### ISSUE TITLE
Rake Tasks for activerecord (up/down) broken?

### ISSUE THREAD

**Issue by woeps** [padrino/padrino-framework#2134]
#### Do you want to request a *feature* or report a *bug*?
Bug

#### What is the current behavior?
Upon running `padrino rake -T` I get:
```
rake aborted!
SyntaxError: /usr/lib/ruby/gems/2.4.0/gems/padrino-gen-0.14.0.2/lib/padrino-gen/padrino-tasks/activerecord.rb:146: syntax error, unexpected tIDENTIFIER, expecting keyword_end
      desc "Runs the "up" for a given MIGRATION_VERSIO
                        ^
/usr/lib/ruby/gems/2.4.0/gems/padrino-gen-0.14.0.2/lib/padrino-gen/padrino-tasks/activerecord.rb:149: syntax error, unexpected tIDENTIFIER, expecting keyword_end
      desc "Runs the "down" for a given MIGRATION_VERSIO
                          ^
/usr/lib/ruby/gems/2.4.0/gems/padrino-gen-0.14.0.2/lib/padrino-gen/padrino-tasks/activerecord.rb:318: syntax error, unexpected keyword_end, expecting end-of-input
  end
     ^
/run/media/chris/DATA/chris/Projects/Ruby/fms_core_prototype/Rakefile:6:in `<top (required)>'
(See full trace by running task with --trace)
```

#### What is the expected behavior?
Get the list of available Rake-Tasks printed out to the terminal.

#### Which versions of Ruby, Padrino, Sinatra, Rack, OS are you using? Did this work in previous versions?
I just updated Gems to:
* Ruby 2.4.1p111
* Padrino 0.14.0.2
* Sinatra 2.0.0
* Rack 2.0.2
* OS: Linux (Manjaro Linux 17.0.1, Kernel: 4.9.24-1)

#### Solution
The problem seems to lie in `padrino-gen-0.14.0.2/lib/padrino-gen/padrino-tasks/activerecord.rb` (like the errormessage states). More precisely: the used quotation marks
Current code:
```ruby
# padrino-gen-0.14.0.2/lib/padrino-gen/padrino-tasks/activerecord.rb
# Line 145
      desc "Runs the "up" for a given MIGRATION_VERSION."  # "up" is the problem
      task(:up => :skeleton){ migrate_as(:up) }

      desc "Runs the "down" for a given MIGRATION_VERSION."  # "down" is the problem
      task(:down => :skeleton){ migrate_as(:down) }
```
Escaping the quotation marks seems to resolve the issue:
```ruby
      desc "Runs the \"up\" for a given MIGRATION_VERSION."  # "up" is the problem
      task(:up => :skeleton){ migrate_as(:up) }

      desc "Runs the \"down\" for a given MIGRATION_VERSION."  # "down" is the problem
      task(:down => :skeleton){ migrate_as(:down) }
```

**Comment by sshaw** [comment]
This was fixed via 4a80dca4ec0c52a9c7daa2e28d17c1d8cb024e4b.

**Comment by woeps** [comment]
Oh, I missed that, sorry.
Thanks anyway!
